lib/livebook/hubs.ex

```diff
@@ -294,4 +295,45 @@ defmodule Livebook.Hubs do
   def verify_notebook_stamp(hub, notebook_source, stamp) do
     Provider.verify_notebook_stamp(hub, notebook_source, stamp)
   end
+
+  @doc """
+  Gets a list of file systems for given hub.
+  """
+  @spec get_file_systems(Provider.t()) :: list(FileSystem.t())
+  def get_file_systems(hub) do
+    hub_file_systems = Provider.get_file_systems(hub)
+    local_file_system = Livebook.Config.local_file_system()
+
+    Enum.sort([local_file_system | hub_file_systems])
```

Comment:
This sort will depend on the order of struct fields, which is not deterministic on OTP 26+. We need to pick an attribute and sort by it. I would also always have the local file system as first. :)

Reply:
I think the local file system should only belong to the personal hub. Depending on how it impacts different places, perhaps it should not belong to any hub at all.

Reply:
It should belong to all hubs because, when we show a file selector inside a notebook, we want the local file system to always be included.

Reply:
Ah nevermind, this is the top-level function, that makes sense yeah.